docker/docker-compose.yml

```diff
@@ -4,13 +4,12 @@ version: "3"
 services:
   relay_alice:
     container_name: alice
-    image: "parity/polkadot:v0.9.42"
+    image: "parity/polkadot:v0.9.43"
     platform: linux/x86_64
     hostname: relay_alice
     ports:
       - "30335:30335" # p2p port
-      - "9935:9933" # rpc port
```

Comment:
we can route both ports to `9944` as an alternative. please let me know which one is more preferable